packages/core/src/command/consolidated/search/fhir-resource/search-consolidated.ts

```diff
@@ -13,3 +36,151 @@ export type SearchConsolidatedResult = {
 export interface SearchConsolidated {
   search({ patient, query }: SearchConsolidatedParams): Promise<SearchConsolidatedResult>;
 }
+
+/**
+ * Performs a search on a patient's consolidated resources in OpenSearch/OS
+ * and returns the resources stored in the OS results.
+ *
+ * @param patient The patient to search.
+ * @param query The query to search for.
+ * @returns The search results.
+ */
+export async function searchPatientConsolidated({
```

Comment:
Moved from `packages/core/src/command/consolidated/search/fhir-resource/search-lexical.ts` w/ modifications.

To see the changes, check out [this commit](https://github.com/metriport/metriport/pull/3940/commits/4436f773ff30bdc252fd6bdd6842814ed1a04722#diff-452c5965401ef361fd077fb55fe1ba087f6427d6a4e735dcde2e085998a56baa).